cmake/libigl.cmake

```diff
@@ -397,8 +397,12 @@ if(LIBIGL_WITH_OPENGL_GLFW_IMGUI)
     if(NOT TARGET imgui)
       igl_download_imgui()
       add_subdirectory(${LIBIGL_EXTERNAL}/libigl-imgui imgui)
+      igl_download_imguizmo()
+      add_library(igl_opengl_glfw_imguizmo ${LIBIGL_EXTERNAL}/imguizmo/ImGuizmo.cpp ${LIBIGL_EXTERNAL}/imguizmo/ImGuizmo.h)
+      add_library(igl::opengl_glfw_imguizmo ALIAS igl_opengl_glfw_imguizmo)
     endif()
     target_link_libraries(igl_opengl_glfw_imgui ${IGL_SCOPE} igl_opengl_glfw imgui)
+    target_link_libraries(igl_opengl_glfw_imguizmo ${IGL_SCOPE} igl_opengl_glfw imgui)
```

Comment:
To simplify things, I would suggest changing the cmake files as follows:
```cmake
    compile_igl_module("opengl/glfw/imgui")
    if(NOT TARGET imgui)
      igl_download_imgui()
      add_subdirectory(${LIBIGL_EXTERNAL}/libigl-imgui imgui)
    endif()
    if(NOT TARGET imguizmo)
      igl_download_imguizmo()
      add_library(imguizmo ${LIBIGL_EXTERNAL}/imguizmo/ImGuizmo.cpp ${LIBIGL_EXTERNAL}/imguizmo/ImGuizmo.h)
    endif()
    target_link_libraries(igl_opengl_glfw_imgui ${IGL_SCOPE} igl_opengl_glfw imgui imguizmo)
```

Reply:
Thank you for the advice- though for it to build I had to also link imgui to the imguizmo target through a `PUBLIC` interface, sort of like the imgui change we needed to make before.
Also, I noticed that imguizmo imports `malloc.h` which breaks on macOS, and someone noted this relatively recently on their github page https://github.com/CedricGuillemet/ImGuizmo/issues/113 without acknowledgement...

Reply:
Can we use an older SHA1 commit for now? One that doesn't break on macOS.